Sources/Utilities/UIApplication+Extension.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  UIApplication+Extension.swift
+//  KommunicateChatUI-iOS-SDK
+//
+//  Created by sathyan elangovan on 07/02/23.
+//
+
+import Foundation
+
+public extension UIApplication {
+    static var main: UIApplication? { UIApplication.value(forKeyPath: #keyPath(UIApplication.shared)) as? UIApplication }
```

Comment:
Won't it still throw a warning if we are using `UIApplication.shared`? We can mark the method as unavailable:
```
@available(iOSApplicationExtension, unavailable, message: "UIApplication.shared is unavailable in application extensions")
```

Reply:
if we make this unavailable to extensions, they can't use this in app-extension target right ? Also they are using this same approach in their app. @mukesh-komm

Reply:
This method won't be directly used from extensions, so it should not throw any error. I am not sure, check once in sample app if possible.

Reply:
changed to function & made it unavailable for app-extension